docs/index.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: nf-validation
+title: nf-schema
 description: Nextflow plugin for sample sheet validation
 ---
 
-# ![nf-validation](images/nf-validation.svg)
+# ![nf-schema](images/nf-schema.png)
```

Comment:
Switch to `png` from `svg` why?

Reply:
Because of the font, that way it looks the same for everyone :p